client/components/date-range/test/index.js

```diff
@@ -297,225 +253,202 @@ describe( 'DateRange', () => {
 				};
 			} );
 
-			const wrapper = shallow( <DateRange translate={ translate } moment={ moment } /> );
-			const datePicker = wrapper.find( DatePicker );
+			const selectedStartDate = moment( '10-01-2018', 'MM-DD-YYYY' );
+			const selectedEndDate = moment( '11-01-2018', 'MM-DD-YYYY' );
 
-			expect( datePicker.props().numberOfMonths ).toEqual( 1 );
-		} );
-
-		test( 'should disable dates before firstSelectableDate when set', () => {
-			const today = new Date();
-			const wrapper = shallow(
-				<DateRange translate={ translate } moment={ moment } firstSelectableDate={ today } />
-			);
-			const datePicker = wrapper.find( DatePicker );
-
-			const expected = [
-				{
-					before: today,
-				},
-			];
-
-			const actual = datePicker.props().disabledDays;
-
-			expect( actual ).toEqual( expected );
-		} );
-
-		test( 'should disable dates after lastSelectableDate when set', () => {
-			const today = new Date();
-			const wrapper = shallow(
-				<DateRange translate={ translate } moment={ moment } lastSelectableDate={ today } />
+			render(
+				<DateRange
+					selectedStartDate={ selectedStartDate }
+					selectedEndDate={ selectedEndDate }
+					translate={ translate }
+					moment={ moment }
+				/>
 			);
-			const datePicker = wrapper.find( DatePicker );
-
-			const expected = [
-				{
-					after: today,
-				},
-			];
 
-			const actual = datePicker.props().disabledDays;
+			await userEvent.click( screen.getByLabelText( 'Select date range' ) );
+			const startMonthEl = screen.getByText( 'October 2018' );
+			const endMonthEl = screen.queryByText( 'November 2018' );
 
-			expect( actual ).toEqual( expected );
+			expect( startMonthEl ).toBeVisible();
+			expect( endMonthEl ).toBeNull();
```

Comment:
Perhaps `.not.toBeInTheDocument()` could be expressing our intentions more precisely than `toBeNull()`

Reply:
Yeah, agreed 👍🏻

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/63879/commits/b50bb71c4c0bb51094e7b6e4ef076ba3ac5489d1.